### PR TITLE
Fixed target framework when compiling WPF solution

### PR DESCRIPTION
**Pull request by alvesmiguel1**

### CODE REVIEW

File: CefGlue.Common/build/CefGlue.Common.props [hunk 1]
```diff
@@ -5,12 +5,14 @@
     </PropertyGroup>
     
     <PropertyGroup Condition="'$(RuntimeIdentifier)' == '' and $([MSBuild]::IsOSPlatform('Windows'))">
+        <CleanTargetFramework>$([System.Text.RegularExpressions.Regex]::Replace($(TargetFramework), '-windows$', ''))</CleanTargetFramework>
```
[the remark this answers is not in the document]
Comment (alvesmiguel1) [reply CefGlue.Common/build/CefGlue.Common.props:8]: But that would require consumers of CefGlue.Common nuget package to define that property. TargetFramework already exists in MSBuild

[the remark this answers is not in the document]
Comment (joaompneves) [reply CefGlue.Common/build/CefGlue.Common.props:8]: we're manually appending the windows suffix here: https://github.com/OutSystems/CefGlue/blob/583a2b9a5fff4b1de876980390e97b0bff1cbc76/CefGlue.WPF/CefGlue.WPF.csproj#L4

File: CefGlue.Demo.WPF/CefGlue.Demo.WPF.csproj [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
     <PropertyGroup>
         <OutputType>WinExe</OutputType>
+        <OperatingSystemAgnosticTargetFramework>$(DefaultTargetDotnetVersion)</OperatingSystemAgnosticTargetFramework>
         <TargetFramework>$(DefaultTargetDotnetVersion)-windows</TargetFramework>
```
Comment (joaompneves) [inline CefGlue.Demo.WPF/CefGlue.Demo.WPF.csproj:6]:
```suggestion
        <TargetFramework>$(DefaultTargetDotnetVersion)-$(OperatingSystemSuffix)</TargetFramework>
```

File: CefGlue.WPF/CefGlue.WPF.csproj [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     
   <PropertyGroup>
+    <OperatingSystemAgnosticTargetFramework>$(DefaultTargetDotnetVersion)</OperatingSystemAgnosticTargetFramework>
     <TargetFramework>$(DefaultTargetDotnetVersion)-windows</TargetFramework>
```
Comment (joaompneves) [inline CefGlue.WPF/CefGlue.WPF.csproj:5]:
```suggestion
    <TargetFramework>$(DefaultTargetDotnetVersion)-$(OperatingSystemSuffix)</TargetFramework>
```

File: CefGlue.Common/build/CefGlue.Common.props [hunk 4]
```diff
@@ -19,8 +21,12 @@
...
     </PropertyGroup>
-    
+
+    <PropertyGroup Condition="'$(OperatingSystemAgnosticTargetFramework)' == ''">
+        <OperatingSystemAgnosticTargetFramework>$(TargetFramework)</OperatingSystemAgnosticTargetFramework>
+    </PropertyGroup>
+
     <ItemGroup Condition="'$(CefGlueTargetPlatform)' != ''">
-        <CefGlueBrowserProcessFiles Include="$(MSBuildThisFileDirectory)..\bin\$(TargetFramework)\$(CefGlueTargetPlatform)\*" />
+        <CefGlueBrowserProcessFiles Include="$(MSBuildThisFileDirectory)..\bin\$(OperatingSystemAgnosticTargetFramework)$(OperatingSystemSuffix)\$(CefGlueTargetPlatform)\*" />
```
Comment (joaompneves) [inline CefGlue.Common/build/CefGlue.Common.props:30]: is this correct? browser process is not windows specific

Comment (alvesmiguel1) [reply CefGlue.Common/build/CefGlue.Common.props:30]: you're right

**Review by joaompneves — APPROVED** [review]

**Pull request merged**